modules/alloy/src/main/ts/ephox/alloy/ui/types/ModalDialogTypes.ts

```diff
@@ -59,7 +59,7 @@ export interface ModalDialogApis {
   show: (dialog: AlloyComponent) => void;
   hide: (dialog: AlloyComponent) => void;
   getBody: (dialog: AlloyComponent) => AlloyComponent;
-  getFooter: (dialog: AlloyComponent) => AlloyComponent;
+  getFooter: (dialog: AlloyComponent) => Optional<AlloyComponent>;
```

Comment:
This needed an Alloy changelog 🤔

Reply:
Added in follow-up PR https://github.com/tinymce/tinymce/pull/8867